### PR TITLE
CCPA and Schain support

### PR DESCRIPTION
**Pull request by HuddledMasses**
## Type of change

- [x] Feature

## Description of change

Addedd ccpa support and supply chains

### CODE REVIEW

**Comment by jsnellbaker** [comment]
Hi @HuddledMasses 

If you haven't already, please ensure you submit a PR to the docs repo to update your bidder file ([here](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders)) to support CCPA.  You just need to add the following variable to the top of your page:
`usp_supported: true`

Thanks.

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/1643

**Comment by HuddledMasses** [comment]
Commented code removed

File: modules/colossussspBidAdapter.js [hunk 1]
```diff
@@ -63,6 +63,12 @@ export const spec = {
       'placements': placements
     };
 
+    if(bidderRequest){
```
Comment (idettman) [inline modules/colossussspBidAdapter.js]: circleci is failing on a linting error from this line: [github-build-link](https://circleci.com/gh/HuddledMasses/Prebid.js/33?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

Fix by changing line 66 to: `if (bidderRequest) {`

Comment (HuddledMasses) [reply modules/colossussspBidAdapter.js]: done

**Pull request merged**